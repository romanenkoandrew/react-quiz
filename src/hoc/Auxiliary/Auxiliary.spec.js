import React from "react";
import { shallow } from "enzyme";
import Auxiliary from "./Auxiliary";

describe("AnswerItem component", () => {
  it("should render AnswerItem", () => {
    const component = shallow(<Auxiliary props={<div>test</div>} />);
    expect(component).toMatchSnapshot();
  });
});
