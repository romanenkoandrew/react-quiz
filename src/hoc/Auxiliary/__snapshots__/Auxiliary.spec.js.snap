// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnswerItem component should render AnswerItem 1`] = `""`;
